### PR TITLE
Add media queries to container elements, update README 

### LINKED ISSUE #60: Mobile-friendly poem.computer
`poem.computer` on a mobile browser is a little wonky, could use some css media queries to change size based on smaller screens 📱

### PR DESCRIPTION
**Pull request by amanda**
(partially at least) resolves #60 
also sort of #48

### app
- add media queries for small screens to most page `container` classes to make them wider so there's no weird overflow
- add placeholder emoji favicon

### docs
- add instructions to install npm dependencies before spinning up docker

## still todo
- i couldn't get poems n stuff in so i almost definitely missed some css fixes, i think this may be related to where the `dev` branch is at? once i can see more of the app i can add more css fixes :D not sure if we want to wait to merge this til then...

### CODE REVIEW

File: client/src/components/login/forms.js [hunk 1]
```diff
@@ -12,24 +11,25 @@ const renderTextField = ({input, label, meta: {touched, error}, ...custom}) => (
...
 
 const loginForm = props => {
   const { handleSubmit } = props
-  
+
   return (
     <form onSubmit={handleSubmit} name='login-form'>
       <div>
-        <Field name="username" component={renderTextField} type="text" placeholder="username"/>
+        <Field name='username' component={renderTextField} type='text' placeholder='username' />
```
Comment (connorwalsh) [inline client/src/components/login/forms.js]: !!!!! yes !!!!!

File: client/src/components/login/forms.js [hunk 2]
```diff
@@ -12,24 +11,25 @@ const renderTextField = ({input, label, meta: {touched, error}, ...custom}) => (
...
       </div>
       <div>
-        <Field name="password" component={renderTextField} type="password" placeholder="password"/>
+        <Field name='password' component={renderTextField} type='password' placeholder='password' />
       </div>
-      <button className='signin-button' type="submit">login</button>
-    </form> 
+      <button className='signin-button' type='submit'>login</button>
```
Comment (connorwalsh) [inline client/src/components/login/forms.js]: as you can tell, my linter isdn't on lol (sorry)

File: client/src/components/login/index.css [hunk 3]
```diff
@@ -47,3 +47,15 @@
     padding-top: 1em;
     color: red;
 }
+
+/* ----------- MOBILE MEDIA QUERIES ----------- */
+
+@media screen and (max-width: 800px) {
```
Comment (connorwalsh) [inline client/src/components/login/index.css]: out of curiosity-- where is this `800px` value coming from?

**Comment by connorwalsh** [comment]
@amanda i am not sure i follow what you mean in your `todo` section-- do you mean you couldn't sign in? or couldn't populate the db with test poets?

**Comment by connorwalsh** [comment]
also, (lol sorry for the rapid fire comments), if you feel up for it and want to try to incorporate some browser tests related to screen size etc. that would be super radical (though i'm not holding it to you to do that since that isn't top priority here w.r.t. what needs more test coverage atm)

File: client/src/components/login/forms.js (hunk 2 above)
Comment (amanda) [reply client/src/components/login/forms.js]: lol it's fine i just linted whatever files i was in but didn't want to lint everything and end up with a huge changeset--it doesn't really matter :)

File: client/src/components/login/index.css (hunk 3 above)
Comment (amanda) [reply client/src/components/login/index.css]: i read this article https://developers.google.com/web/fundamentals/design-and-ux/responsive/ where it says "create breakpoints based on content, never on specific devices, products, or brands." -- 800px seemed to make most of the pages look good when you shrunk it down that much? after that it looked fine wide, also we can always change it! i didn't want a million breakpoints for every possible phone tablet etc. it seemed like bloat.

**Comment by amanda** [comment]
@connorwalsh i will take a look at the signup/user dashboard pages--in my todo i meant i couldn't find the form where you submit a poem i have open in a tab on my phone from saturday that looked weird on mobile, but maybe that's where it is

**Pull request merged**